features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/ApplicationCreation.jsx

```diff
@@ -386,10 +386,9 @@ function ListLabels() {
                         <Typography variant='body2' color='textSecondary' component='p'>
                             <FormattedMessage
                                 id='Workflow.ApplicationCreation.List.empty.content.applicationcreations'
-                                defaultMessage={'There are no workflow pending requests for application creation.'
-                                    + 'It is possible to approve or reject workflow pending requests of application'
-                                    + 'creation. Workflow Approval Executor needs to be enabled to approve or '
-                                    + 'reject the requests. '}
+                                defaultMessage={'There are no pending workflow requests for application creation '+
+                                '(key generation).'
```

Comment:
seems you have changed this as well, which is not correct. Application creation is not equal to the key generation. My comment was only for the application registration below.

Reply:
Fixed it.